client/src/lib/helper.test.js

```diff
@@ -0,0 +1,14 @@
+jest.mock('jquery');
+
+const helper = require('./helper');
+
+describe('client/src/lib/helper', () => {
+  test('should blah', () => {
+    const id = 200;
```

Comment:
Should blah? guessing you forgot to update this test before committing - suggest refactor before merge

Reply:
thank you, will do